library/tiqr/Tiqr/Message/Firebase.php

```diff
@@ -0,0 +1,89 @@
+<?php
+/**
+ * This file is part of the tiqr project.
+ * 
+ * The tiqr project aims to provide an open implementation for 
+ * authentication using mobile devices. It was initiated by 
+ * SURFnet and developed by Egeniq.
+ *
+ * More information: http://www.tiqr.org
+ *
+ * @author Joost van Dijk <joost.vandijk@surfnet.nl>
+ * 
+ * @package tiqr
+ *
+ * @license New BSD License - See LICENSE file for details.
+ *
+ * @copyright (C) 2010-2015 SURFnet BV
```

Comment:
I think the copyright notice should be updated to 2010-2019?

Reply:
Updated